### ISSUE TITLE
Duda M10

### ISSUE THREAD

**Issue by stlara** [IIC2115/Syllabus#187]
En esta pregunta el enunciado dice Output esperado: shape de zonas con información de internet. Esto se refiere al geodataframe de zonas con las columnas viviendas y población u otra cosa?

**Comment by FGarridoV** [comment]
Claro, al geodataframe de zonas, le debes incorporar la nueva información recopilada de internet. Usando el nombre de la comuna como key
